llm_on_ray/inference/api_server_openai.py

```diff
@@ -48,29 +47,14 @@ def router_application(deployments):
 
     RouterDeployment = serve.deployment(
         route_prefix="/",
-        autoscaling_config={
-            "min_replicas": int(os.environ.get("ROUTER_MIN_REPLICAS", 2)),
-            "initial_replicas": int(os.environ.get("ROUTER_INITIAL_REPLICAS", 2)),
-            "max_replicas": int(os.environ.get("ROUTER_MAX_REPLICAS", 16)),
-            "target_num_ongoing_requests_per_replica": int(
-                os.environ.get("ROUTER_TARGET_NUM_ONGOING_REQUESTS_PER_REPLICA", 200)
-            ),
-        },
```

Comment:
Let's remove the autoscaling_config parameters first and then add these parameters to model config yaml files after testing.